docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.8"
```

Comment:
Is this compose file meant to be used by the deployment pipeline? Or just locally?

Reply:
This is just locally for now - It will be used within CI (GitHub actions soon)